components/serializer.rst

```diff
@@ -721,6 +721,15 @@ There are several types of normalizers available:
     :phpclass:`DateTime` and :phpclass:`DateTimeImmutable`) into strings.
     By default it uses the RFC3339_ format.
 
+    The normaliser attempts to denormalize all time strings with a default timezone
+    if the defaultContext sets a `TIMEZONE_KEY` and `FORCE_CONTEXT_TIMEZONE` = true.
```

Comment:
Please use double backticks for all highlightings

Reply:
Done